resource-agents.spec

```diff
@@ -48,7 +48,7 @@
 Name:		resource-agents
 Summary:	Open Source HA Reusable Cluster Resource Scripts
 Version:	3.9.5
-Release:	105%{?dist}.0.0.rdo1
+Release:	105%{?dist}.8.rdo1
```

Comment:
This will not work, NVR in CBS would be:
  resource-agents-3.9.5-105.el7.8.rdo1
and when proposed 7.4.z is released it would not win:
$ rpmdev-vercmp 3.9.5-105.el7_4.9 3.9.5-105.el7.8.rdo1
3.9.5-105.el7_4.9 < 3.9.5-105.el7.8.rdo1

Reply:
To keep track, I suggest  resource-agents-3.9.5-105.el7_4.7.5.rdo
resource-agents-3.9.5-105.el7_4.7 < resource-agents-3.9.5-105.el7_4.7.5.rdo
resource-agents-3.9.5-105.el7_4.8 > resource-agents-3.9.5-105.el7_4.7.5.rdo

Reply:
This will work, to insert temp hotfix in CBS between official el7_4.8 and el7_4.9:
 +Release:       105%{?dist}_4.8.rdo1

Reply:
$ rpmdev-vercmp 3.9.5-105.el7_4.8 3.9.5-105.el7_4.8.rdo1
3.9.5-105.el7_4.8 < 3.9.5-105.el7_4.8.rdo1

$ rpmdev-vercmp 3.9.5-105.el7_4.9 3.9.5-105.el7_4.8.rdo1
3.9.5-105.el7_4.9 > 3.9.5-105.el7_4.8.rdo1